src/main/java/use_cases/display_availability_timeblocks/DisplayAvailabilityTimeBlockController.java

```diff
@@ -0,0 +1,22 @@
+package use_cases.display_availability_timeblocks;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+
+public class DisplayAvailabilityTimeBlockController {
+    private DisplayAvailabilityTimeBlockInputBoundary timeBlockDisplay;
```

Comment:
It's good practice to make any architecture classes like this "final," as they won't change over the course of the program regardless of what is done during its run after they've been initialized.

Reply:
I would go as far as to say, you must do this before you merge; if it's not good enough for IntelliJ then it's definitely not good enough for me :)